snowshu/core/docker.py

```diff
@@ -75,14 +83,14 @@ def get_stopped_container(
         logger.info(f"Created stopped container {container.name}.")
         return container
 
-    def startup(self,
+    def startup(self,   # noqa pylint: disable=too-many-arguments
                 image: str,
                 start_command: str,
                 port: int,
                 target_adapter: str,
                 source_adapter: str,
                 envars: list,
-                protocol: str = "tcp") -> docker.models.containers.Container:
+                protocol: str = "tcp") -> docker.models.containers.Container:   # noqa pylint: disable=unused-argument
```

Comment:
This is interesting too, RN `protocol` is not being used. But did we have plans to support multiple protocol and what difference would it create for the functionality or snowshu?

Reply:
Probably not, but that might have been included due to the TODO message about storages. Should probably just leave the noqa mark, and simplify it later.